src/m365/spo/commands/sitedesign/sitedesign-get.ts

```diff
@@ -72,9 +72,9 @@ class SpoSiteDesignGetCommand extends SpoCommand {
     );
   }
 
-  private getSiteDesignId(args: CommandArgs, spoUrl: string): Promise<string> {
+  private async getSiteDesignId(args: CommandArgs, spoUrl: string): Promise<string> {
     if (args.options.id) {
-      return Promise.resolve(args.options.id);
+      return args.options.id;
     }
 
     const requestOptions: any = {
```

Comment:
along the way lets refactor this to drop `any` and start using `CliRequestOptions`